### DISCUSSION TITLE
[Feature Request] Touch ID support for MacOS?

### DISCUSSION THREAD

**Discussion by ur0** [standardnotes/forum#2524] [Suggestions]
Would it be possible to use Touch ID for wrapping the local passcode for MacOS?

If it would be considered, I'm willing to submit a PR for the same as I've made some progress in implementing it.

**Comment by atmoio** [comment]
I haven't yet looked into this. I wouldn't submit a PR, since something sensitive like this I prefer to develop carefully and with consideration to other features, like Privileges. But please feel free to share your code so I can assess what it involves.

**Comment by myreli** [comment]
It's worth mentioning that when installing the new iOS app on MacOS, the TouchID works as expected. It might be a valid path. 

see https://github.com/standardnotes/forum/issues/1485
Related: https://github.com/standardnotes/forum/discussions/2524

**Comment by PathToLife** [comment]
Is there any update on this?

@ur0 I'm able to help develop, review, or refactor this touch id feature for the latest version of StandardNotes if you would like.

I'll probably check over the iOS app code as it already has biometrics.

**Comment by cdzdavid** [comment]
Any update on this?? It's been now 6 years since the feature was requested... it's a little embarrassing for you guys...
